File: Facebook.NET/GraphAPI.Values.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphAPI
    {
        [DataContract]
        public sealed class Values
        {
            [DataMember(Name = "value")]
            public string Value { get; set; }

            [DataMember(Name = "end_time")]
            public string EndTime { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.Values.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class Values
            <DataMember(Name:="value")>
            Public Property Value() As String
            <DataMember(Name:="end_time")>
            Public Property EndTime() As String
        End Class
    End Class
End Namespace
